hist/hist/inc/v5/TFormula.h

```diff
@@ -237,7 +237,7 @@ class TFormula : public TNamed {
    virtual Int_t       GetNdim() const {return fNdim;}
    virtual Int_t       GetNpar() const {return fNpar;}
    virtual Int_t       GetNumber() const {return fNumber;}
-   virtual TString     GetExpFormula(Option_t *option="") const;
+   virtual TString     GetExpFormula(Option_t *option = "", const char *fl_format = "%g") const;
```

Comment:
The legacy TFormula class should not be changed anymore. Also, the signature of a `virtual` function should not be changed lightly, because people might override it, and if the signature doesn't match the override is ineffective. And if they don't use the `override` keyword to hint to the compiler that it should be an override, they won't even get an error or warning!